src/editor/editorui.js

```diff
@@ -60,8 +54,92 @@ export default class EditorUI {
 		 */
 		this.focusTracker = new FocusTracker();
 
+		/**
+		 * Stores all editable elements used by the editor instance.
+		 *
+		 * @protected
+		 * @member {Array.<module:ui/editorui/editoruiview~EditorUIView>}
+		 */
+		this._editableElements = [];
```

Comment:
Again: `_editableElements` should contains elements. HTML elements, not instances of `EditorUIView`. It should be a map `name (String) -> element (HTMLElement)`. `EditorUIView` or `EditableUIView` usage should be only internal util which can but does not need to, part of the specific editor implementation.